backend/Makefile

```diff
@@ -2,14 +2,10 @@
 -include ../helm-cmd.mk
 HELM_CMD ?= helm upgrade --install
 
-ifndef COMMIT
```

Comment:
if the `COMMIT` env var has a value, it comes from config.yaml and should be used. only if it is empty, the resolution process for the latest image should be used

Reply:
Right, that got lost. I'll add it to the script

Reply:
fixed